Documentation/deploy-addons.md

```diff
@@ -157,5 +157,5 @@ $ kubectl get pods --namespace=kube-system | grep kube-dns-v9
 <div class="co-m-docs-next-step">
   <p>Now that you have a working Kubernetes cluster with a functional CLI tool, you are free to deploy Kubernetes-ready applications.</p>
   <p>Start with a multi-tier web application (Guestbook) from the official Kubernetes documentation to visualize how the various Kubernetes components fit together.</p>
-  <a href="http://kubernetes.io/v1.1/examples/guestbook-go/README.html" class="btn btn-default btn-icon-right" data-category="Docs Next" data-event="kubernetes.io: Guestbook">Deploy the Guestbook Sample app</a>
+  <a href="https://github.com/kubernetes/kubernetes/tree/release-1.1/examples/guestbook-go/" class="btn btn-default btn-icon-right" data-category="Docs Next" data-event="kubernetes.io: Guestbook">Deploy the Guestbook Sample app</a>
```

Comment:
I think we should probably merge this as-is:

The hellonode doc is for creating an app from scratch (and some steps assume a GCE account). The "Kubernetes 101" link in the sidebar ends up linking to the versioned Guestbook example in github (like we have here).

We could link both, e.g. "example building an application from scratch" or "deploy an example guestbook app". But the from-scratch approach alone is going to be more difficult for first time users.

As far as versioning, in some cases we're going to need to keep it tied to our currently targeted version (1.1.X at time of writing).

For example, `kubectl run` functions slightly differently. In v1.2 it will launch a deployment + replica set. In v1.1 it will launch a replication controller.

In this case unless something changed from the last time I tested 1.2 the upstream docs won't actually work as expected (because they document `kubectl get rc` but that will be empty -- because it launches a replicaSet).